tests/test_merge.py

```diff
@@ -43,6 +44,26 @@ def take_first(conflict, _, __, ___):
         self.assertEqual(m.unified_patches,
                          [('change', 'changeme', ('Jo', 'Joe'))])
 
+    def test_continue_run_multiple_conflicts_per_patch(self):
+        lca = {'foo': [{'x': 1}, {'y': 2}]}
+        first = {'foo': [{'x': 1}, {'y': 2}, {'z': 4}]}
+        second = {'bar': 'baz'}
+
+        expected = {
```

Comment:
If you use list of tuples then you don't need following dict access `expected[resolution]` (moreover it's easier to use `pytest.mark.parameterize`).